src/components/Settings/Notifications/NotificationsWebhook/index.tsx

```diff
@@ -111,8 +117,25 @@ const NotificationsWebhook = () => {
       .test(
         'valid-url',
         intl.formatMessage(messages.validationWebhookUrl),
-        isValidURL
+        function (value) {
+          const { supportVariables } = this.parent;
+
+          // allow variable syntax through
+          if (
+            supportVariables &&
+            typeof value === 'string' &&
+            value.includes('{{')
+          ) {
```

Comment:
Shouldn't we just always skip validation if there is `supportVariables` enabled?